modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="update-mirror-repository-oc-mirror_{context}"]
+= Mirroring resources using the oc-mirror plug-in
+
+Using the oc-mirror OpenShift CLI (`oc`) plug-in is the recommended method for mirroring images onto a mirror registry. This can be useful in several situations, including fully or partially disconnected environments.
+
+.Procedure
+
+. Navigate to _Mirroring images for a disconnected installation using the oc-mirror plug-in_.
+
+. Follow the instructions on that page to mirror resources onto a mirror registry.
+
+** If you are using oc-mirror for the first time, follow the instructions on that page up until but not including the section titled *Keeping your mirror registry content updated*.
```

Comment:
I'm a little wary of using a section as where to stop, ahead of it. I did just add a new section to this assembly for 4.12. Luckily it was after "Keeping your...", but if it wasn't, it might have messed your instructions up.

It might be fine to leave it as is (though maybe risky). But I might play with calling out the exact section titles in ordered subbullets. Though I guess they do need to do the prerequisites and should read through the conceptual info, etc. so it might be a lot. I don't know, I'm just brainstorming here :) Give it some thought, but feel free to ignore

Reply:
I see what you mean about future updates potentially complicating these instructions. Part of me hopes that if a new section is created and it's not necessary to the basic workflow, it'll be added near the end of the page like you did with the new section. But I understand that this is not the best assumption to depend on.

I'll try out the bullet list and see how it looks. That said, I personally feel like if someone jams an optional/not-relevant-to-everyone section in the middle of these relevant sections, then in either case it might not be the best user experience to have to follow a bullet list on one page that doesn't exactly replicate the order of sections of the page being referenced. That's just my opinion though, I agree that this is a tricky situation with no great solution.

Reply:
I don't think this entirely addresses the concern you're mentioning here, but I'll change the wording so that it mentions the name of the section to stop **at**, not the one to stop **before**. Still risky but I hope it won't become a problem.